### ISSUE TITLE
Remove create-rdf and create-binary routes in islandora-indexing-fcrepo

### ISSUE THREAD

**Issue by dannylamb** [Islandora-CLAW/CLAW#641]
Blocked by https://github.com/Islandora-CLAW/CLAW/issues/640 and https://github.com/Islandora-CLAW/CLAW/issues/639

The create-rdf and create-binary routes should be merged into the update routes for rdf and binary, respectively.  This depends on Milliner being updated and path minting to be exposed as a microservice.

**Comment by dannylamb** [comment]
Duplicate of https://github.com/Islandora-CLAW/CLAW/issues/640
